### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by KasiaKrakowska**
Review required.

### CODE REVIEW

File: Galway 2020/index.html [hunk 1]
```diff
@@ -96,7 +96,56 @@
...
+						  <button class="btn btn-secondary" onclick="filterSelection('200')"> &euro;0-&euro;200</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('400')"> &euro;201-&euro;400</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('599')">&euro;401-&euro;600</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('over600')"> over &euro;600</button>
+						</div>
+					</div>
+				</div>
+
+
+	</fieldset>
```
Comment (minhtran365411) [inline Galway 2020/index.html]: Category for different price range is working perfectly, all buttons are clickable

File: Galway 2020/index.html [hunk 2]
```diff
@@ -96,7 +96,56 @@
...
+				<li class="filterLi 599"><a class="list" href="#idname">Hotel 2</a><hr><p class="pricetag">&euro;490.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi 200"><a class="list" href="#idname">Hotel 3</a><hr><p class="pricetag">&euro;150.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi over600"><a class="list" href="#idname">Hotel 4</a><hr><p class="pricetag">&euro;450.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi 200"><a class="list" href="#idname">Hotel 1</a><hr><p class="pricetag">&euro;120.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi 400"><a class="list" href="#idname">Hotel 2</a><hr><p class="pricetag">&euro;370.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi 599"><a class="list" href="#idname">Hotel 3</a><hr><p class="pricetag">&euro;470.00<span class="smallsp">/Night</span></p></li>
+				<li class="filterLi over600"><a class="list" href="#idname">Hotel 4</a><hr><p class="pricetag">&euro;550.00<span class="smallsp">/Night</span></p></li>
+			</ul>
 
+	</div>
```
Comment (minhtran365411) [inline Galway 2020/index.html]: Empty cards for different hotels, nothing appears at the moment after clicking on price range.

**Review by minhtran365411 — APPROVED** [review]

File: Galway 2020/index.html (hunk 2 above)
Comment (KasiaKrakowska) [reply Galway 2020/index.html]: Links not working as waiting for final hotel and restaurant list.

**Comment by minhtran365411** [comment]
Finish reviewing!

**Pull request closed without merging**